src/sentry/discover/endpoints/discover_key_transactions.py

```diff
@@ -68,12 +68,16 @@ def get(self, request, organization):
 
         queryset = KeyTransaction.objects.filter(organization=organization, owner=request.user)
 
-        if not queryset.exists():
-            raise ResourceDoesNotExist
-
-        results = key_transaction_query(
-            fields, request.GET.get("query"), params, orderby, "discover.key_transactions", queryset
-        )
+        results = {}
+        if queryset.exists():
```

Comment:
Will do a latter refactor to move this check into the query function, not being done here because I need to remove the conditions param from `discover.query` and cleaner to do those two together.